src/pages/index.js

```diff
@@ -127,20 +182,77 @@ function handleProfileEditClick() {
 }
 
 function handleEditProfileSubmit(inputValues) {
-  userInfo.setUserInfo(profileTitleInput.value, profileDescriptionInput.value);
-  profileEditPopup.close();
+  profileEditPopup.showLoading();
+  api
+    .editProfileInfo(inputValues.title, inputValues.description)
+    .then(() => {
+      userInfo.setUserInfo(inputValues.title, inputValues.description);
+      profileEditPopup.hideLoading();
+    })
+    .catch(console.error)
+    .finally(() => {
+      profileEditPopup.close();
+    });
 }
 
 function handleAddCardSubmit(inputValues) {
-  const newCardData = {
-    name: addCardTitleInput.value,
-    link: addCardImageLinkInput.value,
-  };
-  const newCard = createCard(newCardData);
-  section.addItem(newCard);
-  addCardPopup.close();
+  addCardPopup.showLoading();
+  api
+    .createNewCard(inputValues.title, inputValues.link)
+    .then((res) => {
+      const newCard = createCard(res);
+      cardListSection.addItem(newCard);
+    })
+    .then((res) => {
+      addCardPopup.hideLoading();
+    })
+    .catch(console.error)
+    .finally(() => {
+      addCardPopup.close();
+    });
 }
 
 function handleCardClick(cardData) {
   previewImagePopup.open(cardData);
 }
+
+function handleDeleteBinClick(cardID) {
+  deleteCardPopup.open();
+  deleteCardPopup.setSubmitAction(() => {
+    api
+      .deleteCard(cardID)
+      .then((res) => {
+        this.handleCardDelete();
+      })
+      .catch(console.error)
+      .finally(() => {
+        deleteCardPopup.close();
+      });
+  });
+}
+
+function handleConfirmButtonSubmit() {
+  handleCardDelete();
+}
+
+function handleCardLike(card) {
+  if (card.cardIsLiked()) {
+    api
+      .deleteCardLike(card._cardID)
```

Comment:
`_cardID`

Please, delete the **low dash** from methods if you want to use them `outside` a class because the `low dash` shows a **private** method which is used only inside the class.